### PR TITLE
Refactor scripts in scripts/

### PR DESCRIPTION
**Pull request by ericcornelissen**
**Issue:** n/a
**Alexa rank:** n/a

### Checklist
  - [n/a] I updated the JSON data in `_data/simple-icons.json`
  - [n/a] I optimized the icon with SVGO or SVGOMG
  - [n/a] The SVG `viewbox` is `0 0 24 24`

### Description

While working on some of my recent PRs I noticed some "problems" in our scripts so I tried to fix them all collectively here. Of course these changes are open to discussion. Below is a non-exhaustive list of changes. I would be open to setting up [Prettier](https://prettier.io/) in this Pull Request, as has been discussion on occasion in different PRs.

- Rename `titleToFilename` to `titleToSlug`
- Fix indentation where necessary (to 2 spaces)
- Use quotes internally consistently (to reduce the diff size, unfortunately this is the opposite quote from what we use in other projects)
- Update comments & documentation
- Construct file paths properly using `require('path')`
- Reduce duplication
- Add some missing semicolons

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
All looks good to me, Eric 👍🏻 Given the files it deals with, though, and, in particular with the change in name to `titleToFilename`, I'll let one of the other @simple-icons/maintainers throw a second eye over it before merging.

**Review by mondeja — APPROVED** [review]
LGTM :+1:

**Pull request merged**
